addons/website_slides/models/slide_slide.py

```diff
@@ -706,7 +706,9 @@ def copy_data(self, default=None):
         """Sets the sequence to zero so that it always lands at the beginning
         of the newly selected course as an uncategorized slide"""
         default = dict(default or {})
-        default['sequence'] = 0
+        seen_map = self._context.get('__copy_data_seen', {})
+        if 'slide.channel' not in seen_map:
+            default['sequence'] = 0
```

Comment:
As suggested by Aurélien, why not setting default['sequence'] to 0 only if sequence is not set in defaults ?

Reply:
Because at this point of duplicating a course or a slide, we are not setting the sequence in the defaults.

Reply:
I meant something like: 
```python
def copy_data(self, default=None):
    if 'slide.channel' not in self._context.get('__copy_data_seen', {}):
        if default is None or 'sequence' not in default:
            default = dict(default or {})
            default['sequence'] = 0
    return super().copy_data(default=default)
```
which avoid overriding default if the sequence was explicitly set in the default. 
What do you think ?

Reply:
Ah sorry, we can do that :)